network/p2p/validators_test.go

```diff
@@ -296,6 +296,31 @@ func TestValidatorsTop(t *testing.T) {
 				nodeID1,
 			},
 		},
+		{
+			name: "top ignores inactive validators",
```

Comment:
Can we add a similar test for `TestValdiatorsSample`?

Reply:
`Sample` already ignores disconnected validators and it's unexpected for `EmptyNodeID` to be marked as connected.

So, we could add that case, but it felt a bit odd to me.

If you want it added I'm fine adding it.